gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/model/GenericNotificationConfig.java

```diff
@@ -18,11 +18,17 @@
 import java.util.Date;
 import java.util.List;
 import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NoArgsConstructor;
 
 /**
  * @author Nicolas GERAUD (nicolas.geraud at graviteesource.com)
  * @author GraviteeSource Team
  */
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Why not ```@Getter``` & ```@Setter```?

Reply:
Because there were already defined and there is already a lot of changes 🙈 But I can do the change ;)